pkgs/development/libraries/mimetic/default.nix

```diff
@@ -11,7 +11,13 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ cutee ];
 
-  patches = lib.optional stdenv.isAarch64 ./narrowing.patch;
+  patches = [
+    # Fix build with gcc11
+    (fetchpatch {
+      url = "https://github.com/tat/mimetic/commit/bf84940f9021950c80846e6b1a5f8b0b55991b00.patch";
+      sha256 = "sha256-1JW9zPg67BgNsdIjK/jp9j7QMg50eRMz5FsDsbbzBlI=";
+    })
+  ] ++ lib.optional stdenv.isAarch64 [ ./narrowing.patch ];
```

Comment:
```suggestion
    ./narrowing.patch
  ];
```

Reply:
I tried this, but the `./narrowing.patch` doesn't work outside of Aarch64 (at least not in x86_64), so I fixed the `optional` call but kept the conditional.